src/Products/Metadata/Controllers/MetadataApiController.cs

```diff
@@ -26,20 +26,21 @@ namespace GroupDocs.Metadata.MVC.Products.Metadata.Controllers
     [EnableCors(origins: "*", headers: "*", methods: "*")]
     public class MetadataApiController : ApiController
     {
+        private static List<string> SupportedImageFormats = new List<string>() { ".bmp", ".jpeg", ".jpg", ".tiff", ".tif", ".png", ".gif", ".emf", ".wmf", ".dwg", ".dicom", ".djvu" };
```

Comment:
Codacy found an issue: [Remove these redundant parentheses.](https://app.codacy.com/gh/groupdocs-metadata/GroupDocs.Metadata-for-.NET-MVC/pullRequest?prid=5989453)